projects/1.16/assets/fantasy-mounts/fantasy/lang/zh_cn.json

```
@@ -0,0 +1,23 @@
+﻿{
+  "fantasy.passenger.instruction": "按下 %s 来缩小",
+  "fantasy.whistle.registered": "标记到 %s，将在使用口哨时跟随你（必须拿着口哨）",
+  "fantasy.whistle.come": "%s，过来！",
+  "fantasy.whistle.stay": "%s，停下。",
+
+  "entity.fantasy.unicorn": "独角兽",
+  "entity.fantasy.polar_bear": "北极熊",
+
+  "item.fantasy.unicorn_spawn_egg": "独角兽刷怪蛋",
+  "item.fantasy.polar_bear_spawn_egg": "北极熊刷怪蛋",
+  "item.fantasy.diamond_carrot": "钻石胡萝卜",
+  "item.fantasy.diamond_carrot.desc": "可用于繁殖独角兽",
+  "item.fantasy.golden_carrot_stew": "金胡萝卜煲",
+  "item.fantasy.golden_carrot_stew.desc": "可用于定位独角兽，据说在接近时会使他们发光",
+  "item.fantasy.golden_whistle": "金口哨",
+  "item.fantasy.golden_whistle.tagged": "%s 听从于 %s 的命令。 独角兽只会在你拿着口哨时听从你",
+  "item.fantasy.golden_whistle.not_tagged": "没有标记到任何坐骑。右击标记",
+  "item.fantasy.golden_whistle.following": "跟随中",
+  "item.fantasy.golden_whistle.staying": "休息中",

```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |
```suggestion
  "item.fantasy.golden_whistle.staying": "原地不动中",
```

---

斟酌了一下 因为Mod中的独角兽用处与原版mc中的马等生物类似 因此将状态设为Staying的场景大多是骑乘独角兽结束后 结合现实考虑独角兽（？）此时有可能会感到劳累 所以翻译成“休息中”似乎更符合使用场景且不会有生硬的感觉 再加上原模组文件内的繁中译本对这一条的翻译也是“休息中” 所以自认“休息中”已是比较合适的译名 故这条如果没有更合适的我就先不打算更改了（）

---

结合 stay 的词义和游戏内表现（发出该指令后坐骑确实是一动不动），我认为不需要翻译成“休息”。

> [intransitive](North American English also)to continue to be in a particular place for a period of time without moving away
> \+ adv./prep. **Stay** there and don't move!
> *Oxford Learner's Dictionary*

---

言之有理的 已修改